src/components/UpdateArticle.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import categoryList from "../modules/category";
+import { Container, Grid, Form, Dropdown, Button } from "semantic-ui-react";
+import Preview from "./Preview";
+
+const UpdateArticle = (props) => {
+  const categories = categoryList();
+  return (
+    <Grid columns={2}>
+      <Grid.Column id="left">
+        <Container>
+          <Preview selectedArticle={props.selectedArticle} />
```

Comment:
You guys are not using redux for this? Why passing the article around in props?

Reply:
Beyond my paygrade. We give it a try thou...